ui/Gruntfile.js

```diff
@@ -109,6 +109,7 @@ module.exports = function (grunt) {
         'externalPncUrl': getOpt('pnc-rest-url', 'PNC_UI_PNC_REST_URL', 'https://localhost:9000/pnc-rest/v2'),
         'externalDaUrl': getOpt('da-url', 'PNC_UI_DA_URL'),
         'externalBifrostUrl': getOpt('bifrost-url', 'PNC_UI_BIFROST_URL'),
+        'externalBifrostRestUrl': getOpt('bifrost-rest-url', 'PNC_UI_BIFROST_REST_URL'),
```

Comment:
Under that circumstance, as you mentioned, this will be a non-consistent change that the environment variable will not be compatible with the previous version of PNC UI. Options are:

1. Keep the original variable name for the old variable, and use a new name for the new variable(like what I have implemented to keep the consistency compatibility), or
2. Totally rename the existing variable name to `externalBifrostWsUrl` and add a new variable `externalBifrostRestUrl`,  
3. Refactor current code to make it use a non-protocol server url, like ```"externalBifrostPath": "bifrost-...." // without wss or https prefix``` I suggest we not to use the same variable name as the original one (`externalBifrostUrl`), because this will cause potential compatibility issue while upgrading.

Either we take option 2 or 3, we will need to keep the environment variable `externalBifrostUrl` during our deployment process so that the system would behave the same during our upgrade.

In summary, we have a lot of environments, and our source code repositories are always decoupling with the environment variables (for safety reasons), which brings us a lot of inconsistency, and personally, I would suggest option 1 to minimize any potential impacts during upgrades. After all, we are now focused on WebUI React : )

Reply:
Refactored to ```externalBifrostUrl = https://bifrost.psi.redhat.com
bifrostWsUrl = wss://bifrost.psi.redhat.com/socket```

Reply:
Original state:
externalBifrostUrl ---> websockets

Current state:
externalBifrostUrl ---> http service
bifrostWsUrl ---> websockets

I have 2 questions:
1) so `externalBifrostUrl` is still used, but it's used for http service instead of websockets, am I right?
2) did you updated all use-cases where where `externalBifrostUrl` was originally was used to reflect it's now pointing to http service?

Reply:
1. yes `externalBifrostUrl` now changed to https services.
2. yes, all usages are updated and tested